### ISSUE TITLE
Configure Clippy to be pedantic

### ISSUE THREAD

**Issue by tombruijn** [tombruijn/lintje#37]
This prints a lot more warning about things I should maybe look at.

```
// src/main.rs
#![warn(clippy::pedantic)]
```

> Note that clippy::pedantic contains some very aggressive lints prone to false positives.
> https://github.com/rust-lang/rust-clippy

**Comment by tombruijn** [comment]
Also look at `#![warn(clippy::cargo)]`
